Makefile

```diff
@@ -2,38 +2,49 @@
 publish: release ## alias for `make release`
 
 .PHONY: release
-release: ## Publish a new release
+release: cronjob_scripts/.python-deps-updated.timestamp ## Publish a new release
 	(cd cargo-quickinstall/ && cargo release patch --execute --no-push)
 	git push origin HEAD:release --tags
 	make recheck
```

Comment:
I think `$(MAKE) recheck` is better/more canonical way of invoking another makefile?